bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/PanelImpl.java

```diff
@@ -141,4 +142,9 @@ public Map<String, Object> getDorProperties() {
         return customDorProperties;
     }
 
+    @PostConstruct
+    protected void initGetName() {
```

Comment:
You can add it in `AbstractFormComponentImpl` inside `initBaseModel` since this would be required for all form components. Please add test cases.